src/sentry/snuba/metrics/fields/snql.py

```diff
@@ -319,6 +320,41 @@ def failure_count_transaction(org_id, metric_ids, alias=None):
     )
 
 
+def http_error_count_transaction(org_id, metric_ids, alias=None):
+    statuses = [
+        resolve_tag_value(UseCaseID.TRANSACTIONS, org_id, status)
+        for status in constants.HTTP_SERVER_ERROR_STATUS
+    ]
+    base_condition = Function(
+        "in",
+        [
+            Column(
+                name=resolve_tag_key(
+                    UseCaseID.TRANSACTIONS,
+                    org_id,
+                    TransactionTagsKey.TRANSACTION_HTTP_STATUS_CODE.value,
+                )
+            ),
+            list(status for status in statuses if status is not None),
+        ],
+    )
+
+    return Function(
+        "countIf",
```

Comment:
Shouldn't this be `sumIf` instead? Both raw counters and the counters inside distributions store aggregates, so those have to be added up.

Reply:
As far as I can tell this consistent with the other version of this metric.